docs/emoji-key.md

```diff
@@ -24,14 +24,14 @@ Emoji/Type | Represents | Comments
 📋 <br /> `eventOrganizing` | Event Organizers | Links to event page |
 💵 <br /> `financial` | Financial Support | People or orgs who provide financial support, links to relevant page
 🔍 <br /> `fundingFinding` | Funding/Grant Finders | People who help find financial support
-🤔 <br /> `ideas` | Ideas & Planning | |
+🤔 <br /> `ideas` | Ideas & Planning | Open up issues |
 🚇 <br /> `infra` | Infrastructure | Hosting, Build-Tools, etc. Links to source file (like `travis.yml`) in repo, if applicable
 🚧 <br /> `maintenance` | Maintenance | People who help in maintaining the repo, links to commits by the user on this project
 🧑‍🏫 <br /> `mentoring` | Mentoring | People who mentor new contributors, links to the repo home
 📦 <br /> `platform` | Packaging | Porting to support a new platform |
 🔌 <br /> `plugin` | Plugin/utility libraries | Links to the repo home
-📆 <br /> `projectManagement` | Project Management | |
-📣 <br /> `promotion` | Promotion | |
+📆 <br /> `projectManagement` | Project Management | Team meetings, take minutes, timelines, etc. |
```

Comment:
Could this be worded in a similar tense to the other descriptions?

Reply:
changed from "Team meetings, take minutes, timelines, etc." 
to: "Organize team meetings, take minutes, set timelines, etc."

Reply:
Cool, the commit and PR title should respect the Angular commit convention.